README.md

```diff
@@ -185,9 +186,28 @@ If a published post on openliberty.io/blog contains an error or needs updating i
 
 ## Troubleshooting Asciidoc
 
-Certain characters (eg apostrophe ' ) in the main heading are displayed incorrectly. To fix, escape with a backslash (`\`).
+* Certain characters (eg apostrophe ' ) in the main heading are displayed incorrectly. To fix, escape with a backslash (`\`).
 eg `= Minimise turnaround times with Open Liberty\'s dev mode`
 
+* Syntax highlighting in code snippets isn't displaying correctly. To fix, make sure you've put your code snippet in `source` tags:
+
+   ```
+   [source,xml]
+   ----
+   your code snippet here
+   ----
+   ````
+   Where `xml` is the language used in your code snippet. See the [supported list of languages](https://docs.asciidoctor.org/asciidoctor/latest/syntax-highlighting/coderay/). If your languages isn't supported (eg a Dockerfile or shell script), remove the language attribute: `[source]`.
+
+* Screenshots look scruffy. To fix, place the custom border attribute on the line above the image tag for _all screenshots_:
```

Comment:
Hi @kinueng pinged me w/r/t "scruffy" :) . Probably not a big deal for internal instructions as most will understand the gist but we could also say something like

> For optimal presentation of screenshots in your blog posts, add the `.imageblock.img_border_light` custom border attribute on the line before the image tag for _all screenshots_.

Reply:
Sure. Though I'd tried to phrase as problem followed by solution to be parallel to the other bullets. But I'm fine with anything really so long as it's clear.

Reply:
Yes- I proposed that before I saw it was in a troubleshooting framework. Probably better to keep it parallel. If we want to avoid "scruffy", could say

* Screenshots look awkward or untidy. For optimal presentation of screenshots in your blog posts, add the .imageblock.img_border_light custom border attribute on the line before the image tag for all screenshots.


but scruffy kind of nails it, tbh :)

Reply:
I pinged @dmuelle to help us avoid any possibility that scruffy can be interpreted in a bad way (e.g. prohibited terms).

I am fine with whatever you all decide here so long as it is compliant with words we are allowed to use.

Reply:
It's not a prohibited term but unusual to see in docs so good to check